Java/cryptography/des/DESController.java

```diff
@@ -0,0 +1,153 @@
+package des;
+
+import des.exception.ValidationException;
+import des.key.KeyGenerationModels;
+
+/**
+ * <h1>Data Encryption Standard</h1>
+ * <p>
+ * 		To Implement Data Encryption Standard Symmetric Key Cryptography algorithm with 64-bit plain text and 64-bit cipher
+ * 		text
+ * </p>
+ * <b>Description</b>
+ * <p>
+ * 		class DESController is a public class which acts as a controller, it is used to call cryptography methods of
+ * 		DESCryptography class, the DES algorithm is the most widely used encryption algorithm in the world, DES has been
+ * 		found vulnerable/breakable against many attacks, reflecting it's a decline in popularity but despite this DES will
+ * 		live in many government and banking for years, due to its successor algorithms Triple-DES, AES, etc.
+ * </p>
+ * <b>Reference for understanding the algorithm</b>
+ * <p>
+ * 		<a href="http://page.math.tu-berlin.de/~kant/teaching/hess/krypto-ws2006/des.htm"> The DES Algorithm Illustrated </a>
+ * </p>
+ */
+public class DESController
+{
+	/**
+	 * This is the main method which makes use of the following methods:
+	 * <ul>
+	 * 		<li>DESCryptographyLogic.encrypt</li>
+	 * 		<li>DESCryptographyLogic.decrypt</li>
+	 * </ul>
+	 * 
+	 * And uses the following objects of types:
+	 * <ul>
+	 * 		<li>DESCryptographyLogic</li>
+	 * </ul>
+	 * 
+	 * @param args not used
+	 */

```

Comment:
What is the purpose of using these tags here?

Reply:
It is used maintain a javadoc, it just tells which all methods and objects are used in main method, if anything incorrect, please do let me now.

Reply:
Please remove them up. We are not maintaining a Javadoc here

Reply:
Okay @HarshCasper I'll do that